sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/geojson/GeoBoundingBox.java

```diff
@@ -119,11 +133,48 @@ public boolean equals(Object obj) {
         }
 
         GeoBoundingBox other = (GeoBoundingBox) obj;
-        return west == other.west
-            && south == other.south
-            && east == other.east
-            && north == other.north
+        return Double.compare(west, other.west) == 0
+            && Double.compare(south, other.south) == 0
+            && Double.compare(east, other.east) == 0
+            && Double.compare(north, other.north) == 0
             && Objects.equals(minAltitude, other.minAltitude)
             && Objects.equals(maxAltitude, other.maxAltitude);
     }
+
+    /**
+     * Accesses the coordinates of the {@link GeoBoundingBox} as if it were in a JSON representation.
+     *
+     * @param i Index to access.
+     * @return The double value of the index.
+     * @throws IndexOutOfBoundsException If the {@link GeoBoundingBox} doesn't have altitude coordinates and {@code i}
+     * is greater than {@code 3} or {@link GeoBoundingBox} has altitude coordinates and {@code i} is greater than
+     */
+    public double get(int i) {
+        if (minAltitude != null && maxAltitude != null) {
+            switch (i) {
+                case 0: return west;
+                case 1: return south;
+                case 2: return minAltitude;
+                case 3: return east;
+                case 4: return north;
+                case 5: return maxAltitude;
+                default: throw logger.logExceptionAsWarning(new IndexOutOfBoundsException());
+            }
+        } else {
+            switch (i) {
+                case 0: return west;
+                case 1: return south;
+                case 2: return east;
+                case 3: return north;
+                default: throw logger.logExceptionAsWarning(new IndexOutOfBoundsException());
```

Comment:
Including the index `i` in the exception message might be useful for troubleshooting.